package.json

```diff
@@ -21,6 +21,7 @@
     "typescript": "^4.9.3"
   },
   "dependencies": {
+    "kafkajs": "^2.2.3",
```

Comment:
it should be in devDependencies I think

Reply:
@kornys This is the outcome of installing kafkajs dependency via `npm install kafkajs --save`

Reply:
@asanzgom you should still move it. You should have installed it with `--save-dev`

Reply:
ok, let me do the change @paoloantinori

Reply:
it doesn't makes much of a difference actually ... as in this repo we are only running tests ...

Reply:
change committed

Reply:
@paoloantinori @andreaTP @kornys Can I resolve this conversation? Thank you

Reply:
I'm good